src/renderer/components/SelectMediaBlock.tsx

```diff
@@ -1,10 +1,17 @@
-import { Box, colors, styled, Typography } from '@mui/material';
-import AddIcon from '@mui/icons-material/Add';
+import { Box, styled, Typography, Stack } from '@mui/material';
 import { Dispatch, SetStateAction } from 'react';
+import colors from 'renderer/colors';
+import { extractFileNameWithExtension } from 'renderer/util';
+import ActionButton from './ActionButton';
 
 const SelectMediaBox = styled(Box)`
-  background: ${colors.grey[400]};
-  color: ${colors.grey[900]};
+  width: 100%;
+`;
+
+const InnerBox = styled(Box)`
+  border-style: dashed;
+  border-color: ${colors.grey[500]}
+  color: ${colors.white};
```

Comment:
I know this colour isn't even showing up as white but the text colour here is grey 300

Reply:
Fixed, again, it was the semicolon issue.